packages/config/src/discovery/starkware/getSHARPVerifier.ts

```diff
@@ -12,7 +12,7 @@ const SHARP_VERIFIER_PROXY = discovery.getMainContractDetails(
 
 const SHARP_VERIFIER = discovery.getMainContractDetails(
   'SHARPVerifier',
-  'Starkware SHARP verifier used collectively by StarkNet, Sorare, Immutable X, Apex, Myria and rhino.fi. It receives STARK proofs from the Prover attesting to the integrity of the Execution Trace of these Programs including correctly computed L2 state root which is part of the Program Output.',
+  'Starkware SHARP verifier used collectively by StarkNet, Sorare, ImmutableX, Apex, Myria, rhino.fi and Canvas Connect. It receives STARK proofs from the Prover attesting to the integrity of the Execution Trace of these Programs including correctly computed L2 state root which is part of the Program Output.',
```

Comment:
future idea: get this one automatically from discovery

Reply:
Yep, definietly